### ISSUE TITLE
Is this repository still active?

### ISSUE THREAD

**Issue by Tscheppe** [alexzhirkevich/compose-cupertino#75]
Hi,

Thank you for creating and sharing this fantastic library for Compose Multiplatform! It's been incredibly helpful, and I truly appreciate the effort and thought that went into building it.

I noticed there hasn’t been much recent activity, so I wanted to check if the project is still being actively maintained.

Thank you again for your hard work on this library—it’s greatly valued by the community!

**Comment by schott12521** [comment]
Yes, this repo is currently active -- I am actively trying to get changes merged in.

**Comment by Tscheppe** [comment]
I have seen your contributions and its great that the community participates in this project. However, if the maintainer cannot not review or publish contributions it is still stale in my opinion.

**Comment by begali** [comment]
The maintainer is also contributing.

**Comment by schott12521** [comment]
> I have seen your contributions and its great that the community participates in this project. However, if the maintainer cannot not review or publish contributions it is still stale in my opinion.

Yeah I feel your frustration; I've been trying to update this repo but the lack of communication from the repo owner has been tough. I don't expect immediate responses, but I am trying to make meaningful contributions and without access/responses, it becomes difficult.

If the repo owner needs more support or doesn't want to be the sole maintainer, then I'd like to have a conversation about maintaining this repo without fracturing the community.

**Comment by Tscheppe** [comment]
Since for me the library does not lack any features it isn't really frustrating. I have seen a lot of really cool libraries over the time that became inactive/deprecated. And since the last release is from April I am just a little worried that this might also be the case here. 

> If the repo owner needs more support or doesn't want to be the sole maintainer, then I'd like to have a conversation about maintaining this repo without fracturing the community.

That's what i wanted to find out with this issue. If the maintainer needs help, and if there are plans for this library where external contributors can help.

**Comment by Tweener** [comment]
Is there a list of what's left to do for a new release?

**Comment by MrJohn10** [comment]
@alexzhirkevich

**Comment by schott12521** [comment]
I would like to keep working together if possible, but that is not possible when my comments / PRs / issues go untouched -- and I fully understand that supporting a large open source project is time consuming.

To that end, I think some steps to help mitigate the manual overload for this project/repo include:

- Automate builds / releases (this is huge)
- Automated testing (doesn't have to be in depth yet, just does the app install / launch on all expected platforms)
- Code / repo cleanup

I've been working on some of these changes here: https://github.com/schott12521/ExampleLibrary (note the name/location is temporary, as if I choose to fork compose-cupertino completely then I will rename). For now I'm only publishing to github packages, but the process is automated including tags/releases. If you want to use compose-cupertino with CMP 1.7+, you can do the following:


Add the following to your settings.gradle.kts, making sure that you have set or provided `mavenUser` and `mavenPassword` to your github username and a Personal Access Token.
```
        maven {
            url = uri("https://maven.pkg.github.com/schott12521/ExampleLibrary")
            credentials {
                username = mavenUser
                password = mavenPassword
            }
        }
```

Then you can depend on the latest version of the library (including my [SwipeBox changes](https://github.com/alexzhirkevich/compose-cupertino/pull/74)), in `libs.versions.toml`:
```
cupertino = "0.1.0"

cupertino = { group = "io.github.alexzhirkevich", name = "cupertino", version.ref = "cupertino" }
cupertino-adaptive = { group = "io.github.alexzhirkevich", name = "cupertino-adaptive", version.ref = "cupertino" }
cupertino-decompose = { group = "io.github.alexzhirkevich", name = "cupertino-decompose", version.ref = "cupertino" }
cupertino-native = { group = "io.github.alexzhirkevich", name = "cupertino-native", version.ref = "cupertino" }
cupertino-icons-extended = { module = "io.github.alexzhirkevich:cupertino-icons-extended", version.ref = "cupertino" }
```

Note that this repo/fork is definitely going to change before I more publicly announce my work.
